tests/purs/failing/RecordLabelOnTypeError.out

```diff
@@ -0,0 +1,28 @@
+Error found:
+in module [33mRecordLabelOnTypeError[0m
+at tests/purs/failing/RecordLabelOnTypeError.purs:8:5 - 8:6 (line 8, column 5 - line 8, column 6)
+
+  Could not match type
+  [33m     [0m
+  [33m  Int[0m
+  [33m     [0m
+  with type
+  [33m        [0m
+  [33m  String[0m
+  [33m        [0m
+
+while matching row label [33mfield[0m
+
```

Comment:
There seems to be an extra newline here? I've yet to check as well but at first glance it seems to be the case...

Reply:
Right on the money!

Reply:
you can run with e.g. `HSPEC_ACCEPT=true stack test --test-arguments="--match compiler"` to automatically write the golden test output for you: https://github.com/purescript/purescript/blob/master/CONTRIBUTING.md#running-tests